src/common/data-modeler-state-service/TableStateActions.ts

```diff
@@ -11,10 +11,10 @@ export class TableStateActions extends StateActions {
     @StateActions.PersistentTableAction()
     public addOrUpdateTableToState({stateService, draftState}: PersistentTableStateActionArg,
                                    table: PersistentTableEntity, isNew: boolean): void {
-        if (isNew) {
-            stateService.addEntity(draftState, table);
-        } else {
-            stateService.updateEntity(draftState, table.id, table);
-        }
+        // if (isNew) {
```

Comment:
@AdityaHegde I want to remove this file entirely, since it updates the persistent table state even if the actual table import fails, which means that the state information about the table is for the failed import. I think it's better to call this explicitly in the action that is called rather than here since this seems like another layer of indirection that doesn't serve us.
1. do you have a recommendation about the approach I'm taking or am I misunderstanding the intent?
2. if I delete the whole function `addOrUpdateTableToState`, the tests hang, and I'm not sure why. It should not hang in this case. Any ideas what is happening?